Compilation Error

Board Teensy 3.6

I'm getting the following error compiling the terminalCapabilityTests example.

```
/Applications/Teensyduino.app/Contents/Java/hardware/teensy/../tools/arm/bin/arm-none-eabi-g++ -c -O2 -g -Wall -ffunction-sections -fdata-sections -nostdlib -mno-unaligned-access -MMD -fno-exceptions -fpermissive -felide-constructors -std=gnu++14 -Wno-error=narrowing -fno-rtti -mthumb -mcpu=cortex-m4 -mfloat-abi=hard -mfpu=fpv4-sp-d16 -fsingle-precision-constant -D__MK66FX1M0__ -DTEENSYDUINO=154 -DARDUINO=10815 -DARDUINO_TEENSY36 -DF_CPU=180000000 -DUSB_SERIAL -DLAYOUT_US_ENGLISH -I/var/folders/0q/kx24shts5396rqt4r2rgpkfr0000gn/T/arduino_build_949051/pch -I/Applications/Teensyduino.app/Contents/Java/hardware/teensy/avr/cores/teensy3 -I/Users/patricklittle/Documents/Arduino/libraries/retroTerm/src /var/folders/0q/kx24shts5396rqt4r2rgpkfr0000gn/T/arduino_build_949051/sketch/Example01_terminalCapabilityTests.ino.cpp -o /var/folders/0q/kx24shts5396rqt4r2rgpkfr0000gn/T/arduino_build_949051/sketch/Example01_terminalCapabilityTests.ino.cpp.o
In file included from /Users/patricklittle/Documents/Arduino/libraries/retroTerm/examples/Example01_terminalCapabilityTests/Example01_terminalCapabilityTests.ino:20:0:
/Users/patricklittle/Documents/Arduino/libraries/retroTerm/src/retroTerm.h:754:11: error: 'uint8_t retroTerm::newTextInput(uint8_t, uint8_t, uint8_t, uint8_t)' cannot be overloaded
   uint8_t newTextInput(const uint8_t x, const uint8_t y, const uint8_t w, const uint8_t h)
           ^
/Users/patricklittle/Documents/Arduino/libraries/retroTerm/src/retroTerm.h:700:11: error: with 'uint8_t retroTerm::newTextInput(uint8_t, uint8_t, uint8_t, uint8_t)'
   uint8_t newTextInput(uint8_t x, uint8_t y, uint8_t w, uint8_t h)
           ^

--- a/src/retroTerm.h
+++ b/src/retroTerm.h
@@ -697,7 +697,7 @@ class retroTerm
 		#if defined(ESP8266) || defined(ESP32)
 		uint8_t ICACHE_FLASH_ATTR newRadioButton(uint8_t x, uint8_t y, uint8_t w, uint8_t h)
 		#else
-		uint8_t newTextInput(uint8_t x, uint8_t y, uint8_t w, uint8_t h)
+		uint8_t newRadioButton(uint8_t x, uint8_t y, uint8_t w, uint8_t h)
 		#endif
 		{
 			return(newWidget(_widgetTypes::textInput, x, y, w, h, _defaultAttributes, _defaultStyle));

